config/config.go

```diff
@@ -152,4 +144,13 @@ func init() {
 		}
 		Cfg.SolanaIndexerWorkers = parsedWorkers
 	}
+
+	contentNodeMonitorInterval := os.Getenv("contentNodeMonitorInterval")
```

Comment:
imho we don't add any value by having this exposed via env versus just hard-coded in the file. feels unnecessary / more error prone to do this way, but whatever!

Reply:
I'm happy to remove it and hard-code it. My thought was that it would make it easier to write a test for this were I so inclined. But I'll yank it for now